mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -813,438 +840,538 @@ class vehicles {
+
+A NOTE ON ORDERING -------------------------------------------------------------------
+
+each class is sorted in the following order
+
+1. whitelisting (unlocked first, then public-ish, then WLUs alphabetically)
+2. inherited from tempalte above first, then custom spawners
+3. ground -> air
+4. light -> heavy
+
+please try to stick to that order in future so it's easy to find things!
+*/
+
+
+class spawn_point_types {
+
+	// unlocked /////////////////////////////////////////////////////////////////////////
+
+	// spike team base helos
+
+	class unlocked_air_transport_light: parent_air_transport_light {
+		RESPAWN_SHORT;
+		lockTeams[] = UNLOCKED;
 	};
 
+	// acav base
 
-	class utility {
-		name = "Utility";
+	class unlocked_grnd_patrol: parent_grnd_patrol {
+		RESPAWN_MEDIUM;
+		lockTeams[] = UNLOCKED;
+	};
+	class unlocked_grnd_light_transport: parent_grnd_light_transport {
+		RESPAWN_SHORT;
+		lockTeams[] = UNLOCKED;
+	};
+	class unlocked_grnd_transport_trucks: parent_grnd_transport_trucks {
+		RESPAWN_LONG;
+		lockTeams[] = UNLOCKED;
+	};
+	class unlocked_grnd_utility: parent_grnd_utility {
 		RESPAWN_SHORT;
+		lockTeams[] = UNLOCKED;
+	};
+
+	// boat dock
 
+	class unlocked_boats_small {
+		name = "Small Boats (Public)";
+		RESPAWN_SHORT;
+		lockTeams[] = UNLOCKED;
 		class categories {
-			class fuel {
-				name = "Fuel";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { {"fuel" }};
-				exclude[] = { {"airport" }};
+			class pbr {
+				name = "PBR";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "naval", "pbr" } };
+				exclude[] = { { "pavn" } };
 			};
-			class ammo {
-				name = "Ammo";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { {"ammo" }};
-				exclude[] = { {"airport" }};
+			class boat {
+				name = "Stab";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "naval", "stab" } };
+				exclude[] = { { "pavn" } };
 			};
-			class repair {
-				name = "Repair";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { {"repair" }};
-				exclude[] = { {"airport" }};
+			class RHIB {
+				name = "RHIB";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "naval" , "RHIB"} };
+				exclude[] = { { "pavn" } };
 			};
 		};
 	};
 
-	class utility_airport {
-		name = "Airport Utility";
-		RESPAWN_SHORT;
+	// green hornets /////////////////////////////////////////////////////////////////////
 
-		class categories {
-			class fuel {
-				name = "Fuel";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "fuel", "airport" } };
-				exclude[] = { { "pavn" } };
-			};
-			class ammo {
-				name = "Ammo";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "ammo", "airport" } };
-				exclude[] = { { "pavn" } };
-			};
-			class repair {
-				name = "Repair";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "repair", "airport" } };
-				exclude[] = { { "pavn" } };
-			};
-		};
+	class greenhornets_air_transport_all: parent_air_transport_all {
+		RESPAWN_MEDIUM;
+		lockTeams[] = LOCKED_PUBLIC_HELOS;
 	};
```

Comment:
no implementation of this, there is a template for it though  `parent_grnd_utility_airport`